ang/civicase/CaseListTable.html

```diff
@@ -42,7 +42,7 @@
       civicase-sortheader="{{ header.sort }}">{{ header.label }}</th>
   </tr>
   </thead>
-  <tbody>
+  <tbody class="civicase__table--caselist__body">
```

Comment:
Is this class used anywhere? Remove if not.

Reply:
Used to add border to the left.

Reply:
Removed